wemake_python_styleguide/violations/consistency.py

```diff
@@ -1799,23 +1799,11 @@ class RedundantSubscriptViolation(ASTViolation):
     Example::
```

Comment:
Can you please substitute the whole example section with:

```
    Example::

        # Correct:
        array[:7]
        array[3:]

        # Wrong:
        x[0:7]
        x[3:None]
```

Reply:
Oops, that was my bad. Will do.

Reply:
Looks like the example is still not fix. I'll help you with this one.